docs/dictionary/command/reset.lcdoc

```diff
@@ -35,7 +35,7 @@ number of objects of that type with the property settings you want.
 >*Note:* The LiveCode development environment makes changes to template
 > objects when it creates new objects. This means that objects you
 > create in the development environment may not have the same property
-> settings as objects you create in a <a/>standalone application.
+> settings as objects you create in a standalone application.
```

Comment:
This whole document could do with reference peppering.
- 32, Properties, template, objects
- 33, objects
- 35, template
- 36, objects x3
- 38, objects & standalone application